primitives/src/lib.rs

```diff
@@ -343,6 +343,12 @@ pub type UnsignedFixedPoint = FixedU128;
 /// The `Inner` type of the `UnsignedFixedPoint`.
 pub type UnsignedInner = u128;
 
+pub trait CurrencyInfo {
```

Comment:
This is not used? I would prefer to have the declaration of the trait and addition of the actual usage in the same PR